### ISSUE TITLE
Accessibility: Layout Editor should be implemented as tables and buttons

### ISSUE THREAD

**Issue by jquesada2016** [Dygmalab/Bazecor#1047]
**Describe the bug**

Currently, the Layout Editor seems to be implemented linearly, using a bunch of lists, as my screen reader reports. Both the Dygma keyboard as well as the reference keyboard should be tables, with each halve of the Dygma keyboard having it's own keyboard. This would allow screenreaders to be able to navigate switches much easier and quicker.

There are also a lot of headings thrown in places where they should be buttons instead, and makes navigation with the screen reader quite confusing.

**To Reproduce**
1. Open your favorite screen reader (JAWS, NVDA, or VoiceOver)
2. Open the Layout Editor
3. Try navigating the screen.
4. Notice the dificulty of navigation.

**Expected behavior**

I expect editing the layout to be less torturous for screen reader users.

**Screenshots**
If applicable, add screenshots to help explain your problem. If possible, please also include a log from the "developer console". You can access it by pressing `Control + Shift + I`. When it appears, right click inside it, and select "Save as", and attach the saved file to this report.

**Desktop (please complete the following information):**
 - OS: Linux, macOS, Windows
 - Bazecor Version: 1.7.0

**Additional context**

**Comment by MiquelDygma** [comment]
You're right, Bazecor is not completely polished for screen readers. I'll add this to the backlog.
